webview/src/plots/components/ribbon/RibbonBlockTooltip.tsx

```diff
@@ -42,6 +42,9 @@ export const RibbonBlockTooltip: React.FC<{
           ))}
         </tbody>
       </table>
+      <p className={styles.tooltipTableDescription}>
+        *Reflects the first three columns (minus Created) in the table
```

Comment:
`*Reflects the first three columns (minus Created) in the table`

- Could swap `*` with another info icon. 
- Change `minus` to `excluding`.
- Need to mention that it is the experiments table.
- Give instructions on how to update. Something like "Drag and drop columns to the start of the table to update.

This could also go after the commit information. 

I'm almost tempted to suggest turning the ribbon into a mini version of the table that always displays the selected columns but I think that is a story in itself.

Reply:
> Need to mention that it is the experiments table.
> Change minus to excluding.

👍 

> Give instructions on how to update. Something like "Drag and drop columns to the start of the table to update.

I considered that but didn't want to make the message too verbose since it's already rather lengthy... I'll post an image and see what others think though. 

> Could swap * with another info icon.

Good idea, I'll try it and see how it looks! 

> This could also go after the commit information.

Why would we want to move the table below the commit information? I figured that the table information is more important than the commit hence why it's first 🤔

Reply:
Here's what it looks like with extra instructions:

<img width="304" alt="Screenshot 2023-06-20 at 11 00 04 AM" src="https://github.com/iterative/vscode-dvc/assets/43496356/8429428b-9468-48e9-bc60-10e6e85480d7">

It felt way too lengthy to me. Though I suppose if we increase the table rows, that could change 🤔